docs/src/understand/mqtt-bus.md

```diff
@@ -1,7 +1,52 @@
 ---
 title: MQTT Bus
 tags: [Concept, MQTT]
-sidebar_position: 1
+sidebar_position: 3
 ---
 
-# The Thin Edge MQTT bus
+# Thin Edge MQTT bus
+
+Thin-edge uses a combination of MQTT and HTTP to coordinate the interactions
+between the miscellaneous hardware and software components that make up a piece of equipment.
+
+- A local MQTT broker is deployed along each thin-edge enabled piece of equipment.
+- Each device and service is given dedicated MQTT topics to publish their measurements, events and alarms
+  as well as to receive targeted operation requests.
+- Similarly, a local HTTP server is used to transfer files to and from child devices,
+  using a URL schema that is aligned with the schema used to specifically target devices over MQTT. 
+
+Collectively, the MQTT broker and the normalized MQTT topic schema form the __MQTT bus__.
+The local HTTP server and the normalized URLs schema are referred as the __file transfer service__.
+
+## Local Digital Twin
+
+The MQTT topic schema establishes the local representation of a piece of equipment, with its main device, child devices and services.
+This local representation has a one-to-one relationship with the remote digital twin.
+However, while the latter is a user interface the former is akin to a program interface, an API.
+
+- the main device as well as the child devices are each represented by a topic
+- each type of telemetry data, measurements, events, alarms, is given a specific sub-topic of the topic related to the source device
+- each type of operations is given a specific sub-topic of the topic related to the target device
+
+## Cloud Bridges
+
+The MQTT bus is extended to connect the cloud using an MQTT bridge
+that publishes back and forth messages from the local MQTT broker to the remoted MQTT endpoint.
+
+Such a bridge is by definition cloud specific - each kind of messages having to be sent or received from specific topics.
+
+## Local File Transfer
+
+Using MQTT is appropriate to collect telemetry data from devices and services
+as well as to trigger operations on devices and services.
+
+However, it's more pertinent to use HTTP to transfer files from one device to another.
+And this is not only because of payload size constraints, but also for *pull* versus *push* constraints.
```

Comment:
```suggestion
as well as to trigger operations on devices and services. However, it's more pertinent to use HTTP to transfer files from one device to another.
This is not only because of payload size constraints, but also for *pull* versus *push* constraints.
```

Reply:
Removed